### DISCUSSION TITLE
Which version should we be installing?

### DISCUSSION THREAD

**Discussion by jcam** [emporia-vue-local/esphome#98] [General]
The instruction guide says to install @dev, but of course that is a moving target. Is it stable enough to just always go with dev, or should I be taking the most recent non-beta tag (currently 2022.11.5)?

Thanks so much for all y'all are doing!!

**Comment by flaviut** [comment]
I'd say it's stable enough, since:

- this is not a fast-moving project, there's [only been one](https://github.com/emporia-vue-local/esphome/pull/90) significant change since around February.
- the risky parts, like the OTA updates, come from the main ESPHome project, not this repo.
- all upgrades are 100% optional & manual on your end.

Of course, you should make your own decision based on your personal risk tolerance :)

**Reply by williamobrien** [reply]
As a related question, I've had this installed for a while, but I moved some circuits around in my panel, which changed their phase. I want to update the device's configuration, but I'm afraid that when I make those changes to the config file, it might break due to some change to ESPHome. Mine's been stable with ESPHome 2022.1.4, but it wants to update to 2022.12.3. Do you have any idea how I can update the config OTA without updating the firmware version of ESPHome?

**Reply by flaviut** [reply]
- From HomeAssistant: I'm not sure, it'll be tricky
- From the command-line: `pip install esphome==2022.12.3`

**Reply by williamobrien** [reply]
Darn, yeah that's what I was thinking too. Maybe I can install the 2022.1.4  version of ESPHome in a virtual machine, make my changes and compile it and do it that way. Hmm. What version of ESPHome has been tested to work with emporia-vue-local?

**Reply by flaviut** [reply]
I updated my system a week or so ago. The Emporia integration is fairly simple, so I don't expect errors that would prevent OTA updates to come up often.

**Reply by williamobrien** [reply]
Thank you! Perhaps I'll just try and go ahead with the newest version as I update it and hope that everything just works since the integration is simple.
